include/daxa/c/device.h

```diff
@@ -165,6 +165,17 @@ typedef struct
     uint32_t invocation_reorder_mode;
 } daxa_RayTracingInvocationReorderProperties;
 
+// Is NOT ABI Compatible with VkPhysicalDeviceHostImageCopyProperties!
+typedef struct
+{
+    uint32_t copy_src_layout_count;
+    daxa_ImageLayout * copy_src_layouts;
```

Comment:
Daxa deprecated all image layouts aside from 'present_src, general and undefined'.
IMO, we should not expose the possible layouts here at all.
I think we should just check if general layout is supported in both src and dst and if not return unsupported